operator/internal/manifests/internal/config/build_test.go

```diff
@@ -159,6 +160,8 @@ storage_config:
       server_address: dns:///loki-index-gateway-grpc-lokistack-dev.default.svc.cluster.local:9095
 tracing:
   enabled: false
+analytics:
+  reporting_enabled: false
```

Comment:
Should we enable the reporting in the test, so that we see the field updated (false=default)?